<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreditWindow</class>
 <widget class="QWidget" name="CreditWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: #00FFFF;
}

</string>
  </property>
  <widget class="QPushButton" name="pushButton_back">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: #FFC0CB;
	color: black;
	border: 1px solid gray;
	font-size: 16px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #DB7093, stop: 1 #FFC0CB);
}</string>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_about">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>370</y>
     <width>300</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: #FFC0CB;
	color: black;
	border: 1px solid gray;
	font-size: 16px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #DB7093, stop: 1 #FFC0CB);
}</string>
   </property>
   <property name="text">
    <string>Что за кнопка такая интересная?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_calculate_credit">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>370</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: #FFC0CB;
	color: black;
	border: 1px solid gray;
	font-size: 16px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #DB7093, stop: 1 #FFC0CB);
}</string>
   </property>
   <property name="text">
    <string>Рассчитать</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_money">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>160</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: black;
}</string>
   </property>
   <property name="text">
    <string>Общая сумма кредита</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_time_r">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: black;
}</string>
   </property>
   <property name="text">
    <string>Срок кредита</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_percent">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>140</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: black;
}</string>
   </property>
   <property name="text">
    <string>Процентная ставка</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_rate_of_payment">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>170</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: black;
}</string>
   </property>
   <property name="text">
    <string>Тип кредита</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_credit_money">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>40</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	color: black;
	border: 1px solid gray;
}</string>
   </property>
   <property name="text">
    <string>100000</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_time_replament">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>100</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	color: black;
	border: 1px solid gray;
}</string>
   </property>
   <property name="text">
    <string>12</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_rate_of_exchange">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>160</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	color: black;
	border: 1px solid gray;
}</string>
   </property>
   <property name="text">
    <string>9.8</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_time">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>100</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
	color: black;
	border: 1px solid gray;
}</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_credit_type">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>220</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
	color: black;
	border: 1px solid gray;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>160</y>
     <width>20</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: black;
}</string>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>40</y>
     <width>30</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: black;
}</string>
   </property>
   <property name="text">
    <string>руб.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_close">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>570</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: #FFC0CB;
	color: black;
	border: 1px solid gray;
	font-size: 16px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #DB7093, stop: 1 #FFC0CB);
}</string>
   </property>
   <property name="text">
    <string>Закрыть</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>400</y>
     <width>600</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: black;
}</string>
   </property>
   <property name="text">
    <string>____________________________________________________________________________________</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>450</y>
     <width>230</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: black;
}</string>
   </property>
   <property name="text">
    <string>Ежемесячный платеж:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_total_monthly_payment">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>450</y>
     <width>200</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: black;
}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>480</y>
     <width>180</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: black;
}</string>
   </property>
   <property name="text">
    <string>Общая выплата:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_total_payment">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>480</y>
     <width>200</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: black;
}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>510</y>
     <width>170</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: black;
}</string>
   </property>
   <property name="text">
    <string>Переплата по кредиту:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_total_above_payment">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>510</y>
     <width>200</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: black;
}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
